README.md

```diff
@@ -25,182 +74,203 @@ repositories {
 apply plugin: 'org.jetbrains.dokka'
 ```
 
-or using the plugins block:
+The plugin adds `dokkaHtml`, `dokkaJavadoc`, `dokkaGfm` and `dokkaJekyll` tasks to the project.
+ 
+Each task corresponds to one output format, so you should run `dokkaGfm` when you want to have a documentation in `GFM` format.
+Output formats are explained [there](#output_formats)
 
-```groovy
-plugins {
-    id 'org.jetbrains.dokka' version '0.10.0'
-}
-repositories {
-    jcenter() // or maven { url 'https://dl.bintray.com/kotlin/dokka' }
-}
-```
-
-The plugin adds a task named `dokka` to the project.
-
-If you encounter any problems when migrating from older versions of Dokka, please see the [FAQ](https://github.com/Kotlin/dokka/wiki/faq).
+If you encounter any problems when migrating from older versions of dokka, please see the [FAQ](https://github.com/Kotlin/dokka/wiki/faq).
 
 Minimal dokka configuration:
 
-Groovy
-```groovy
-dokka {
+Kotlin
+(single-platform project)
+```kotlin
+tasks.dokkaHtml {
     outputDirectory = "$buildDir/dokka"
 }
 ```
-
-Kotlin
+(mutli-platform project)
 ```kotlin
 tasks.dokkaHtml {
     outputDirectory = "$buildDir/dokka"
+    dokkaSourceSets {
+        create("jvmMain")
+        create("jsMain") // or other names, identical to those in Kotlin-plugin
+    }
 }
 ```
-You may need to add an `import org.jetbrains.dokka.gradle.dokka` to the top of `build.gradle.kts` in this case.
+You may need to add an `import org.jetbrains.dokka.gradle.dokka` to the top of `build.gradle.kts`.
```

Comment:
We can remove this sentence: I removed the extension function in favour of gradle automated generated accessors

Reply:
It's still needed for custom tasks, removed though